xmake/core/project/project.lua

```diff
@@ -896,7 +896,8 @@ end
 function project.ordertargets()
     local ordertargets = project._memcache():get("ordertargets")
     if not ordertargets then
-        local targets = project.targets()
+        local targets = table.values(project.targets())
+        table.sort(targets, function (a,b) return a:name() < b:name() end)
```

Comment:
targets is map, it will break

Reply:
I used `table.values` on it to turn it into an array

Reply:
but instance_deps.sort_deps need map, not array.

Reply:
Fixed